test/092_convex_wrapper.ts

```diff
@@ -304,4 +304,19 @@ describe('Convex Wrapper', async function () {
     expect(await convexWrapper.balanceOf(join)).to.eq(posted.add(beforeJoinBalance))
     expect(await fyToken.balanceOf(ownerAcc.address)).to.eq(borrowed.add(beforeFyTokenBalance))
   })
+
+  it('Adding a vault for a different collateral fails', async () => {
+    await wand.makeIlk(USDC, CVX3CRV, compositeMultiOracle.address, 1000000, 1000000, 1, 6)
+    await cauldron.addIlks(seriesId, [CVX3CRV])
+    var join = await ladle.joins(CVX3CRV)
+    await convexWrapper.point(join)
+    // Batch action to build a vault & add it to the wrapper
+    const addVaultCall = convexLadleModule.interface.encodeFunctionData('addVault', [
+      convexWrapper.address,
+      '0x000000000000000000000000',
+    ])
+    await expect(
+      ladle.batch([ladle.buildAction(seriesId, USDC), ladle.moduleCallAction(convexLadleModule.address, addVaultCall)])
+    ).to.be.revertedWith('revert Vault is for different ilk')
```

Comment:
The error message is different:
"Vault is for different ilk"
'revert Vault is for different ilk'

Is this test ok?

Reply:
Really strange that it is passing. Let me investigate.

Reply:
The above test is failing locally. After changing the message to reflect what is in the contract is making it pass.